Code/JavaWrappers/gmwrapper/CMakeLists.txt

```diff
@@ -98,7 +98,7 @@ ADD_CUSTOM_COMMAND(
   COMMAND ${CMAKE_COMMAND} -E make_directory build-test
 
   ## Add in our own Java sources
-  #COMMAND ${COPY_CMD}
+  COMMAND ${COPY_CMD}
```

Comment:
I think Lipinski.java is vestigial and should be removed.

The reason I would like to avoid copying over .java files in general is because it makes the wrappers java specific (well, it makes it the capabilities of the wrappers for different languages different).